notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb

```diff
@@ -0,0 +1,602 @@
+        "plt.xticks(rotation=45)\n",
+        "# label the y axis for clarity\n",
+        "plt.ylabel(\"New Cases\")\n",
+        "\n",
+        "# Show the plot\n",
+        "plt.show()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sM5-HFDx70RG"
+      },
+      "source": [
+        "## Visualization #2: Symptom-related searches compared to new cases"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "se1b6Vf4XB9_"
+      },
+      "source": [
+        "### Filter data"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Wl2o-NYMoygb"
+      },
+      "source": [
+        "We're curious if searches for symptoms like \"cough\" and \"fever\" went up in the same times and places that new COVID-19 cases occured, compared to non-symptoms like \"bruise.\" Let's plot searches vs. new cases to see if it looks like there's a correlation."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "olfnCzyg8jYi"
+      },
+      "source": [
+        "First, we select the new cases column and the search trends we're interested in."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "LqqHzjty8jk0"
+      },
+      "outputs": [],
+      "source": [
+        "symptom_data = all_data[[\"new_confirmed\", \"search_trends_cough\", \"search_trends_fever\", \"search_trends_bruise\"]]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b3DlJX-k9SPk"
+      },
+      "source": [
+        "Not all rows have data for all of these columns, so let's select only the rows that do."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "g4MeM8Oe9Q6X"
+      },
+      "outputs": [],
+      "source": [
+        "symptom_data = symptom_data[\n",
+        "    symptom_data[\"search_trends_cough\"].notna() &\n",
+        "    symptom_data[\"search_trends_fever\"].notna() &\n",
+        "    symptom_data[\"search_trends_bruise\"].notna() &\n",
+        "    symptom_data[\"new_confirmed\"].notna()\n",
+        "]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "IlXt__om9QYI"
+      },
+      "source": [
+        "We want to use a line of best fit to make the correlation stand out. Matplotlib does not include a feature for lines of best fit, but seaborn, which is built on matplotlib, does.\n",
+        "\n",
+        "BigQuery DataFrames does not currently integrate with seaborn by default. So we will demonstrate how to downsample and download a DataFrame, and use seaborn on the downloaded data."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "MmfgKMaEXNbL"
+      },
+      "source": [
+        "### Downsample and download"
```

Comment:
I thought we don't encourage our users to download the data?

Reply:
Much of the time, yes, but downloading and downsampling for visualization was specifically requested for an example.